joint_state_publisher/joint_state_publisher/joint_state_publisher.py

```diff
@@ -59,6 +59,46 @@ def _init_joint(self, minval, maxval, zeroval):
             joint['effort'] = 0.0
         return joint
 
+    def init_sdf(self, robot):
+        robot = robot.getElementsByTagName('model')[0]
```

Comment:
How about SDF files that contain many models?  How should we handle those?

Reply:
I think we should error, and only allow one model

Reply:
I think it should error, and only allow documents with one model.

https://github.com/ros/sdformat_urdf/pull/1/files#diff-12b109e5d429371a684ac656c3aa2037R59-R64

Reply:
SDFormat no longer allows files with multiple root-level models, so I'll resolve this thread.

https://github.com/gazebosim/sdformat/pull/619